cloud-init/configure-falcon-sensor.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+"""Set the customer ID for the CrowdStrike Falcon sensor.
+
+This file is a template.  It must be processed by Terraform.
+"""
+
+from __future__ import annotations
+
+# Standard Python Libraries
+# Bandit triggers B404 here, but we're only using subprocess.run() and
+# doing so safely.  For more details on B404 see here:
+# https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess
+import subprocess  # nosec
+import sys
+from typing import Any
+
+# Third-Party Libraries
+import boto3
+
+# Inputs from Terraform
+FALCON_CUSTOMER_ID_KEY: str = "${falcon_customer_id_key}"
+FALCON_SENSOR_INSTALL_PATH: str = "${falcon_sensor_install_path}"
+FALCON_TAGS_KEY: str = "${falcon_tags_key}"
+SSM_READ_ROLE_ARN: str = "${ssm_read_role_arn}"
+SSM_REGION: str = "${ssm_region}"
+
+
+def get_parameter(ssm_client, parameter_name: str, with_decryption: bool = True) -> str:
+    """Get the value of the specified parameter."""
+    response: dict[str, Any] = ssm_client.get_parameter(
+        Name=parameter_name,
+        WithDecryption=with_decryption,
+    )
+    return response["Parameter"]["Value"]
+
+
+def main() -> int:
+    """Retrieve necessary values from SSM Parameter Store and set the customer ID."""
+    # Create STS client
+    sts_client = boto3.client("sts")
+
+    # Assume the role that can read the SSM Parameter Store parameters
+    stsresponse: dict[str, Any] = sts_client.assume_role(
+        RoleArn=SSM_READ_ROLE_ARN,
+        RoleSessionName="set_crowdstrike_falcon_sensor_customer_id",
+    )
+    newsession_id = stsresponse["Credentials"]["AccessKeyId"]
+    newsession_key = stsresponse["Credentials"]["SecretAccessKey"]
+    newsession_token = stsresponse["Credentials"]["SessionToken"]
```

Comment:
I'm assuming these are the correct type. Since the dictionary is typed to `dict[str, Any]` we should specify what type we expect these variables to hold.

```suggestion
    newsession_id: str = stsresponse["Credentials"]["AccessKeyId"]
    newsession_key: str = stsresponse["Credentials"]["SecretAccessKey"]
    newsession_token: str = stsresponse["Credentials"]["SessionToken"]
```